### PR TITLE
style: fix lint issues in code 🕺

### PR DESCRIPTION
**Pull request by vinayakkulkarni**
- [x] Continued from #596 cause `main` branch was renamed.

Signed-off-by: Vinayak Kulkarni <19776877+vinayakkulkarni@users.noreply.github.com>

### CODE REVIEW

**Comment by acalcutt** [comment]
Is it possible to ignore these errors in. These are purposely empty functions to allow for the 'light' version switch to work.

/opt/test7/vinayakkulkarni/src/serve_light.js
  4:10  warning  'options' is defined but never used       @typescript-eslint/no-unused-vars
  4:19  warning  'repo' is defined but never used          @typescript-eslint/no-unused-vars
  4:28  error    Unexpected empty method 'init'            @typescript-eslint/no-empty-function
  5:9   warning  'options' is defined but never used       @typescript-eslint/no-unused-vars
  5:18  warning  'repo' is defined but never used          @typescript-eslint/no-unused-vars
  5:24  warning  'params' is defined but never used        @typescript-eslint/no-unused-vars
  5:32  warning  'id' is defined but never used            @typescript-eslint/no-unused-vars
  5:36  warning  'publicUrl' is defined but never used     @typescript-eslint/no-unused-vars
  5:47  warning  'dataResolver' is defined but never used  @typescript-eslint/no-unused-vars
  5:64  error    Unexpected empty method 'add'             @typescript-eslint/no-empty-function
  6:12  warning  'repo' is defined but never used          @typescript-eslint/no-unused-vars
  6:18  warning  'id' is defined but never used            @typescript-eslint/no-unused-vars
  6:25  error    Unexpected empty method 'remove'          @typescript-eslint/no-empty-function

File: src/main.js [hunk 1]
```diff
@@ -110,60 +91,72 @@ const startWithMBTiles = (mbtilesFile) => {
...
       process.exit(1);
     }
 
     instance.getInfo((err, info) => {
       if (err || !info) {
         console.log('ERROR: Metadata missing in the MBTiles.');
-        console.log(`       Make sure ${path.basename(mbtilesFile)} is valid MBTiles.`);
+        console.log(
+          `       Make sure ${path.basename(mbtilesFile)} is valid MBTiles.`,
+        );
```
Comment (acalcutt) [inline src/main.js:105]: I still don't like how this breaks up some lines. is this a line length thing?

Comment (vinayakkulkarni) [reply src/main.js:105]: There's space within the ` – https://github.com/maptiler/tileserver-gl/blob/master/src/main.js#L113

Comment (vinayakkulkarni) [reply src/main.js]: I've removed this in [58db9b5](https://github.com/maptiler/tileserver-gl/pull/626/commits/58db9b5703f498e2e45d9e8060bcdeb906ae2e4a)

**Comment by vinayakkulkarni** [comment]
> Is it possible to ignore these errors in. These are purposely empty functions to allow for the 'light' version switch to work.
> 
> /opt/test7/vinayakkulkarni/src/serve_light.js 4:10 warning 'options' is defined but never used @typescript-eslint/no-unused-vars 4:19 warning 'repo' is defined but never used @typescript-eslint/no-unused-vars 4:28 error Unexpected empty method 'init' @typescript-eslint/no-empty-function 5:9 warning 'options' is defined but never used @typescript-eslint/no-unused-vars 5:18 warning 'repo' is defined but never used @typescript-eslint/no-unused-vars 5:24 warning 'params' is defined but never used @typescript-eslint/no-unused-vars 5:32 warning 'id' is defined but never used @typescript-eslint/no-unused-vars 5:36 warning 'publicUrl' is defined but never used @typescript-eslint/no-unused-vars 5:47 warning 'dataResolver' is defined but never used @typescript-eslint/no-unused-vars 5:64 error Unexpected empty method 'add' @typescript-eslint/no-empty-function 6:12 warning 'repo' is defined but never used @typescript-eslint/no-unused-vars 6:18 warning 'id' is defined but never used @typescript-eslint/no-unused-vars 6:25 error Unexpected empty method 'remove' @typescript-eslint/no-empty-function

- [x] Done in 5b5ac87

**Comment by vinayakkulkarni** [comment]
> Looks good to me. I'll likely approve this soon.
> 
> What do you thing we should do with all the lint warnings? Do you think these are something we should work on over time after approving this?

ideally yes! we should fix (at least) the security ones first then move on the fixing the rest imo, this will also help grow the community as it'll be easier to contribute for new folks!

**Review by acalcutt — APPROVED** [review]

**Comment by acalcutt** [comment]
Hey @vinayakkulkarni , I tried merging this into my fork, but it seems to cause my docker build to fail. It looks like it tries to set up husky, but that package is excluded because the docker uses --omit=dev. It gets this error.

```
#15 101.1 npm ERR! code 127
#15 101.1 npm ERR! path /usr/src/app
#15 101.1 npm ERR! command failed
#15 101.1 npm ERR! command sh -c -- husky install
```

guessing we could maybe move husky to the non-dev dependencies... or is there any what not to run that when dev is omitted?

**Comment by vinayakkulkarni** [comment]
> 



> Hey @vinayakkulkarni , I tried merging this into my fork, but it seems to cause my docker build to fail. It looks like it tries to set up husky, but that package is excluded because the docker uses --omit=dev. It gets this error.
> 
> ```
>  > [linux/amd64 builder 5/5] RUN cd /usr/src/app && npm ci --omit=dev:
> #15 101.1 npm notice Changelog: <https://github.com/npm/cli/releases/tag/v8.19.3>
> #15 101.1 npm notice Run `npm install -g npm@8.19.3` to update!
> #15 101.1 npm notice 
> #15 101.1 npm ERR! code 127
> #15 101.1 npm ERR! path /usr/src/app
> #15 101.1 npm ERR! command failed
> #15 101.1 npm ERR! command sh -c -- husky install
> ```
> 
> guessing we could maybe move husky to the non-dev dependencies... or is there any what not to run that when dev is omitted?
> 
> EDIT: heres an example run [acalcutt/tileserver-gl/actions/runs/3426002406/jobs/5707345214](https://github.com/acalcutt/tileserver-gl/actions/runs/3426002406/jobs/5707345214)

Should be fixed in https://github.com/maptiler/tileserver-gl/pull/639

**Pull request merged**